sdk/storage/azure-storage-blob-changefeed/tests/test_change_feed.py

```diff
@@ -333,3 +334,14 @@ def test_list_3_shards_events_works_with_1_shard_cursor(self, storage_account_na
         dict_token_with_1_shard = json.loads(token_with_1_shard)
         self.assertEqual(len(dict_token_with_1_shard['CurrentSegmentCursor']['ShardCursors']), 1)
         self.assertEqual(len(dict_token['CurrentSegmentCursor']['ShardCursors']), 3)
+
+    # @ChangeFeedPreparer()
+    # def test_azure_named_key_credential_access(self, storage_account_name, storage_account_key):
```

Comment:
Test is passing when ran live -- but then doesn't generate a recording. Any idea on how to resolve this? I have checked my `settings_real.py` file and to no avail 😢